### DISCUSSION TITLE
Welcome to infidel-sensor Discussions!

### DISCUSSION THREAD

**Discussion by drspangle** [drspangle/infidel-sensor#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leonhma** [comment]
Hi Just an Idea: Add the link to GitHub Pages in the Info of this repo

**Comment by oskarsjogren** [comment]
Hi @drspangle, I'm very new to all this but wanted to make a sensor to try out on a Ender 3 printer. I have very little experience in electronics and I must say that I'm lost when it comes to the different steps. Also, the soldering of the hall sensor is tricky, not sure I managed that without short circuits...

When I program the card with the calibration.ino the LED lights up, I understand that it should flash twice but this is not the case. I know this is the first step, so if I can't get past this I'm pretty sure I will not be able to finish this sensor that seems pretty cool. :)

I'm doing the programmming with an Arduino UNO by the way.

I would appreciate any hints on what might be going on here. Should I abandon already, or is there hope?

Thanks
Oskar

**Reply by drspangle** [reply]
@oskarsjogren You're in a touch spot. It's hard to know whether you've broken the sensor or done an inadequate soldering job. You should start by taking out your multimeter and testing the voltages along the circuit. Don't give up, first start by making sure that the circuit is correctly built. If you are able to program the unit successfully then you have already made a lot of progress.

**Comment by Skywalker2204** [comment]
Hey @oskarsjogren i was watchin the youtube vid and was impressed by the sensor. I am using klipper as firmware and was wondering if it is possible to run the sensor via the host, withour unsing another micro contorller. It turns out to be pretty simple, by just adapting existing python files. The main davantage here is, no need for any electonics on the senor, just wire to the main printer board. 
I'm still working on it but was also thinking to change the geometry to implement a connector or something. However, by now it is working fine and looking forward to add it to my github, and cann link it if you want. 

Thanks for the grat work 

Lukas
